docs/documentation/tasks/rails/index.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: Custom Rails Tasks
+layout: default
+---
+
+Many of these tasks probably require [Capistrano::Rails](https://github.com/capistrano/rails).
+
+### Run arbitrary rake tasks from environment variables
+
+From [Capistrano/Rails PR #209](https://github.com/capistrano/rails/pull/209)
+
+```ruby
+namespace :ruby do
+  desc 'Runs any rake task, cap deploy:rake task=db:rollback'
+  task rake: [:set_rails_env] do
+    on release_roles([:db]) do
+      within release_path do
+        with rails_env: fetch(:rails_env) do
+          execute :rake, ENV['task']
+        end
+      end
+    end
+  end
+end
+```
+
+Passes in the rake task to be run via an environment variable. Also a simple example of running a rake task on the server.
```

Comment:
Might be good to add an example here like you did below in the migrations section. E.g.

```
bundle exec cap production deploy task=db:seed
```

Aside: does Rake automatically turn key=value arguments into `ENV` values? That is a bit unexpected.

Reply:
Will do. Per Rake handling environment variables, it sure does: https://github.com/ruby/rake/blob/master/lib/rake/application.rb#L741